esp/esdllib/esdl_transformer2.cpp

```diff
@@ -1196,8 +1197,7 @@ void Esdl2Struct::addChildren(Esdl2Transformer *xformer, IEsdlDefObjectIterator
         {
             m_children.append(*obj);
             m_child_map.setValue(obj->queryName(), obj);
-            if (obj->queryEclName())
```

Comment:
ok I'll add ecl_name back. Also I found a similar optimization need for EsdlDefStruct, I'll add the fix to the same pull request (unless Tony or Richard think it's better to open another ticket, but I think it's ok to use the same ticket as it's fixing the same problem in 2 areas).